src/lazy-image.component.ts

```diff
@@ -33,17 +33,20 @@ export class LazyImageComponent implements AfterViewInit, OnInit {
     @Input() public sources: ImageSource[];
     @Input() public visibilityOverride: boolean;
     @Input() public loadingTpl: TemplateRef<any>;
+    @Input() public errorTpl: TemplateRef<any>;
```

Comment:
make sure that errorTpl is optional and component is still working

Reply:
It is, checked.